Drop HHVM from GitHub workflows?

Seems to do not work anyway:

> Run hhvm vendor/bin/phpunit
>
> Fatal error: Not a Hack file in /home/runner/work/bigbluebutton-api-php/bigbluebutton-api-php/vendor/phpunit/phpunit/phpunit on line 1
> Error: Process completed with exit code 255.

(see [Split integration test suite into non-invasive integration and invasive functional suite by FelixJacobi · Pull Request #90 · littleredbutton/bigbluebutton-api-php](https://github.com/littleredbutton/bigbluebutton-api-php/pull/90/checks?check_run_id=2976403331))

HHVM dropped PHP support several years ago. I guess we can drop this job safetly.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -76,17 +76,3 @@ jobs:
         COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       if: ${{ matrix.php-versions == '7.2' && env.COVERALLS_REPO_TOKEN != null }}
       run: vendor/bin/php-coveralls --coverage_clover=build/logs/coverage.xml -v
-
-
-  HHVM:
-    name: HHVM
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - uses: azjezz/setup-hhvm@v1
-        with:
-          version: 'latest'
-      - run: composer install --dev --no-interaction
-        continue-on-error: true
-      - run: hhvm vendor/bin/phpunit
-        continue-on-error: true

